README.md

```diff
@@ -360,6 +360,7 @@ Following options are available for the JS Obfuscator:
     debugProtectionInterval: false,
     disableConsoleOutput: false,
     domainLock: [],
+    exclude: [],
```

Comment:
This option is CLI-only option, so it shouldn't be here. It's already described in the list of CLI-only options.

Reply:
But I think that it's a good idea to add an additional list of CLI-options descriptions

Reply:
But it seems to be working fine with config js file 😐